MultiVersions/Forge/main/plushies/lang/zh_tw.json

```
@@ -1,24 +1,60 @@
-{
-  "block.plushies.panda": "貓熊絨毛玩具",
-  "block.plushies.slime": "史萊姆絨毛玩具",
-  "block.plushies.red_fox": "紅色狐狸絨毛玩具",
-  "block.plushies.white_fox": "白色狐狸絨毛玩具",
-  "block.plushies.dragon": "龍絨毛玩具",
-  "block.plushies.cow": "牛絨毛玩具",
-  "block.plushies.mooshroom": "哞菇絨毛玩具",
-  "block.plushies.bee": "蜜蜂絨毛玩具",
-  "block.plushies.llama": "駱馬絨毛玩具",
-  "block.plushies.magma_cube": "岩漿立方怪絨毛玩具",
-  "block.plushies.axolotl": "六角恐龍絨毛玩具",
-  "block.plushies.turtle": "海龜絨毛玩具",
-  "block.plushies.snow_golem": "雪人絨毛玩具",
-  "block.plushies.dolphin": "海豚絨毛玩具",
-  "block.plushies.parrot": "鸚鵡絨毛玩具",
-  "block.plushies.warden": "伏守者絨毛玩具",
-  "block.plushies.frog": "青蛙絨毛玩具",
-  "block.plushies.goat": "山羊絨毛玩具",
-  "block.plushies.cat": "貓絨毛玩具",
-  "block.plushies.enderman": "終界使者絨毛玩具",
-  "block.plushies.sheep": "羊絨毛玩具",
-  "itemGroup.plushies": "絨毛玩具"
+{
+  "block.plushies.panda_plushie": "貓熊絨毛玩具",
+  "block.plushies.slime_plushie": "史萊姆絨毛玩具",
+  "block.plushies.red_fox_plushie": "紅色狐狸絨毛玩具",
+  "block.plushies.white_fox_plushie": "白色狐狸絨毛玩具",
+  "block.plushies.dragon_plushie": "龍絨毛玩具",
+  "block.plushies.cow_plushie": "牛絨毛玩具",
+  "block.plushies.mooshroom_plushie": "哞菇絨毛玩具",
+  "block.plushies.bee_plushie": "蜜蜂絨毛玩具",
+  "block.plushies.llama_plushie": "駱馬絨毛玩具",
+  "block.plushies.magma_cube_plushie": "岩漿立方怪絨毛玩具",
+  "block.plushies.axolotl_plushie": "六角恐龍絨毛玩具",
+  "block.plushies.turtle_plushie": "海龜絨毛玩具",
+  "block.plushies.snow_golem_plushie": "雪人絨毛玩具",
+  "block.plushies.dolphin_plushie": "海豚絨毛玩具",
+  "block.plushies.parrot_plushie": "鸚鵡絨毛玩具",
+  "block.plushies.warden_plushie": "伏守者絨毛玩具",
+  "block.plushies.frog_plushie": "青蛙絨毛玩具",
+  "block.plushies.goat_plushie": "山羊絨毛玩具",
+  "block.plushies.cat_plushie": "貓絨毛玩具",
+  "block.plushies.enderman_plushie": "終界使者絨毛玩具",
+  "block.plushies.sheep_plushie": "綿羊絨毛玩具",
+  "block.plushies.pig_plushie": "豬絨毛玩具",
+  "block.plushies.tadpole_plushie": "海龜絨毛玩具",
+  "block.plushies.allay_plushie": "悅靈絨毛玩具",
+  "block.plushies.vex_plushie": "惱鬼絨毛玩具",
+  "block.plushies.black_rabbit_plushie": "黑色兔子絨毛玩具",
+  "block.plushies.brown_rabbit_plushie": "棕色兔子絨毛玩具",
+  "block.plushies.killer_bunny_plushie": "殺手兔絨毛玩具",
+  "block.plushies.salt_rabbit_plushie": "'Salt' 兔子絨毛玩具",

```

Minecraft Wiki 上譯為 `棕白色`
```suggestion
  "block.plushies.salt_rabbit_plushie": "'棕白色兔子絨毛玩具",

```